Pkl/StdOutput/MessagePackStandardOutputReader.cs

```diff
@@ -23,22 +24,27 @@ public void Start()
     }
 
     private void BeginRead()
-    {
+    {   
```

Comment:
Looks like we got some extraneous spaces here, but more of a nit pick than something we need to fix.